src/library/scala/collection/immutable/Stream.scala

```diff
@@ -201,6 +201,7 @@ import scala.language.implicitConversions
  *  @define orderDependentFold
  *  @define willTerminateInf Note: lazily evaluated; will terminate for infinite-sized collections.
  */
+@SerialVersionUID(4897901117128916054L)
```

Comment:
Was this added intentionally?  We have a few other PRs about serialisation, so I wouldn't want this flying in by mistake.